homeassistant/components/plugwise/switch.py

```diff
@@ -19,12 +19,24 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
     coordinator = hass.data[DOMAIN][config_entry.entry_id][COORDINATOR]
 
     entities = []
+    switch_classes = ["plug", "switch_group"]
+
     all_devices = api.get_all_devices()
     for dev_id, device_properties in all_devices.items():
-        if "plug" in device_properties["types"]:
-            model = "Metered Switch"
+        members = None
+        model = None
+
```

Comment:
dummy should have a bit more descriptive name

Reply:
Sure, done.